spec/core_ext/array/tableize_spec.rb

```diff
@@ -34,6 +34,17 @@
         expect(test.tableize).to eq(expected)
       end
 
+      it 'removes color' do
+        test = [["Col1", "Col2"], ["Val1", "Val2"], ["\033[31mReally Really Long Value3\e[0m", "Value4"]]
+        expected = <<-EOF
+ Col1                               | Col2
+------------------------------------|--------
+ Val1                               | Val2
+ Really Really Long Value3          | Value4
```

Comment:
I'm surprised this has a lot of right-side padding...I'd expect your change to make it look like the test on line 26.

Reply:
and since your test is passing, then is the PR not working as expected?